1-exercises/E-for-loop/exercise1.js

```diff
@@ -8,7 +8,8 @@
 // Change the below code to use a for loop instead of a while loop.
 let i = 0;
 while(i < 26) {
```

Comment:
Please think over the logic of this solution because we got into the infinite loop and use only the while loop that should replace a for loop.